lms/djangoapps/shoppingcart/tests/test_views.py

```diff
@@ -918,7 +918,12 @@ class mock_get_order(object):  # pylint: disable=invalid-name
         """Mocks calls to EcommerceAPI.get_order. """
         patch = None
 
-        ORDER = copy.deepcopy(EcommerceApiTestMixin.ECOMMERCE_API_SUCCESSFUL_BODY)
+        ORDER = {
+            'status': OrderStatus.COMPLETE,
+            'number': EcommerceApiTestMixin.ORDER_NUMBER,
+            'payment_processor': 'cybersource',
+            'payment_parameters': {'orderNumber': EcommerceApiTestMixin.ORDER_NUMBER},
```

Comment:
I'm sorry, I see that this is meant to mock a serialized order. Assuming this is the case, `payment_processor` and `payment_parameters` are no longer part of serialized orders, so you should be able to remove them.